src/constants/chainInfo.ts

```diff
@@ -23,6 +23,18 @@ import { ARBITRUM_LIST, AVALANCHE_LIST, BASE_LIST, CELO_LIST, OPTIMISM_LIST, PLA
 
 export const AVERAGE_L1_BLOCK_TIME = ms(`12s`)
 
+// The block number at which v3 was deployed on each chain, separate from the UNIVERSAL_ROUTER_CREATION_BLOCK
+export const START_BLOCKS: { [key: number]: number } = {
```

Comment:
taken from https://github.com/Uniswap/v3-info/blob/master/src/constants/index.ts#L52

Reply:
can we request that this be moved into an SDK? for now, just open a linear ticket to track this.

Reply:
https://linear.app/uniswap/issue/WEB-2890/move-start-blocks-into-shared-sdk